src/main/java/org/concordion/internal/ImplementationStatusChecker.java

```diff
@@ -121,6 +121,35 @@ public ResultSummary convertForCache(ResultSummary rs) {
             // if we're expected to pass, then just use the result summary.
             return rs;
         }
+    },
+    SKIPPED(ImplementationStatus.SKIPPED) {
+        @Override
+        public void assertIsSatisfied(ResultSummary rs, FailFastException ffe) {
+            if (rs.getSuccessCount() + rs.getFailureCount() + rs.getExceptionCount() + rs.getIgnoredCount() > 0 || ffe != null) {
+                throw new ConcordionAssertionError("Example is expected to be skipped but is currently reporting.", rs);
+            }
+        }
+
+        @Override
+        public String printNoteToString() {
+            return "   <-- Note: This example has been marked as SKIPPED";
+        }
+
+        @Override
+        public ResultSummary getMeaningfulResultSummary(ResultSummary rs, FailFastException ffe) {
+            assertIsSatisfied(rs, ffe);
+            return new SingleResultSummary(Result.IGNORED);
```

Comment:
So here we have skipped being processed as ignored for the results. So another indicator they are probably the same concept. I've read your comment about the conceptual mismatch with jUnit - it's a good point and I'll have a think about it (woke up early today and brain not yet working!)